pydoctor/model.py

```diff
@@ -1385,3 +1385,35 @@ def buildModules(self) -> None:
         self.system.process()
 
 System.systemBuilder = SystemBuilder
+
+def prepend_package(builderT:Type[ISystemBuilder], package:str) -> Type[ISystemBuilder]:
```

Comment:
maybe add a docstring here as a way to document to other dev what is the low-level purpose of this method.